examples/nfs_file.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * NFS Read and Write - example app
+ * Run the code using `node examples/nfs_file.js` cmd from root directory.
+ * NOTE: Open SAFE Browser instance with the authenticator add-on from the terminal.
+ *       Auth request popups on executing this code.
+ *       Click Allow button and you will see response URI printed on safe_browser terminal.
+ *       Copy that response URI and paste it on example terminal next to `Enter Authenticator response =>`.
+ */
+
+const readline = require('readline');
+const crypto = require('crypto');
+const safeApp = require('../src/');
+const lib = require('../src/native/lib');
+
+
+const appInfo = {
+  'id': 'net.maidsafe.examples.nfsReadWrite',
+  'name': 'NodeJS example App - NFS read/write',
```

Comment:
this breaks on Linux because of https://github.com/maidsafe/system_uri/issues/9 .

But let's fix that post merge.